zktrie/src/state.rs

```diff
@@ -80,6 +80,44 @@ impl ZktrieState {
         true
     }
 
+    ///
```

Comment:
we still need poseidon hashing inside the query_xx method?

Reply:
Yes. The secure key require a poseidon hash.

I am trying to add a "key preimage" feature to skip these hashes

Reply:
done. Now there is no hash calculation on buildin initial zktrie state